brevo/assets/logs/brevo_tests.yaml

```diff
@@ -0,0 +1,1069 @@
+id: brevo
+tests:
+  - sample: |-
+      {
+      "date_sent" : "2020-10-09 00:00:00",
```

Comment:
Looking at the sample now, it seems you are not always providing the `timestamp`, is that intended? Shouldn't we use another property potentially if timestanps is not provided like ts here.

Please note we do handle only mili timestamp in Datadog, but you can transform a second timestamp with a processor into milisecond